### PR TITLE
Update pygen source files

### PR DESCRIPTION
**Pull request by aneels3**
Fix flake8 errors!

### CODE REVIEW

File: pygen/pygen_src/isa/riscv_instr.py [hunk 1]
```diff
@@ -166,14 +168,66 @@ def create_csr_filter(self, cfg):
...
+        else:
+            instr_names_set = set(self.instr_names)
+            disallowed_instr_set = set(disallowed_instr)
+            allowed_instr_set = set(allowed_instr)
+            include_instr_set = set(include_instr)
+            excluded_instr_names_list = list(instr_names_set - disallowed_instr_set)
+            excluded_allowed_instr_list = list(allowed_instr_set - disallowed_instr_set)
+            include_instr_list = list(include_instr_set - disallowed_instr_set)
+
+            name = random.choice(excluded_instr_names_list)
```
Comment (udinator) [inline pygen/pygen_src/isa/riscv_instr.py:213]: I think the approach you use here is ok temporarily, but we will need a more robust system to handle the more complex constraints used elsewhere in the generator.
Can you add a TODO here to switch this randomization method over to whichever solution we decide to go with later on?

Comment (aneels3) [reply pygen/pygen_src/isa/riscv_instr.py:213]: Sure.

**Comment by danghai** [comment]
It LGTM

File: pygen/pygen_src/isa/riscv_instr.py [hunk 2]
```diff
@@ -166,14 +168,67 @@ def create_csr_filter(self, cfg):
...
+            if(len(include_instr) > 0):
+                name = random.choice(include_instr_list)
+            if(len(allowed_instr) > 0):
+                name = random.choice(excluded_allowed_instr_list)
+            if(name is None):
+                logging.critical("%s Cannot generate random instruction", riscv_instr.__name__)
+                sys.exit(1)
+
+        # Shallow copy for all relevant fields, avoid using create() to improve performance
```
Comment (taoliug) [inline pygen/pygen_src/isa/riscv_instr.py:223]: this comment can be removed, it's UVM specific

Comment (aneels3) [reply pygen/pygen_src/isa/riscv_instr.py]: Fixed.

**Pull request merged**